### DISCUSSION TITLE
Is it possible to make an choose an image form?

### DISCUSSION THREAD

**Discussion by tedr56** [Budibase/budibase#12147] [Help]
I'm making a looking up app via a rest api.
When more than one result return I was thinking to display the associated resulted images and you would have to click on the right one you'rer looking for more info.
I was thinking to add this to the first steps of my form. I've manage to display the pictures but I don't have any actions possible with the image component.
Is there any way to achieve this?
Thanks you in advance.

**Comment by ConorWebb96** [comment]
Hey @tedr56,

Could you provide some screenshots of what you have managed so far. I'm trying to visual what you are asking and struggling a bit with purely text. When you mentioned hooking into the rest API are we talking about a Budibase internal db API. Or is this a different rest api somewhere else.

If its a rest api hosted elsewhere I don't believe we can upload or change images currently, you can pass/change the url to the image. 

The image component should be able to display images that are public accessible on the net. Additionally you could do another rest call on click of the image that opens a side panel or a new page about the record.

I hope this makes sense.

If I get some additional information. I could look into providing screenshots/gif of an example.
